### PR TITLE
Fix Windows file path

### PR DESCRIPTION
**Pull request by vlascik**

### CODE REVIEW

**Comment by Turbo87** [comment]
@vlascik this PR seems to fix two separate problems. can you split it into two PRs so that we can discuss them separately? 🙏

**Comment by jelhan** [comment]
@Turbo87 Totally agree that PRs should only target one issue per time. But this is targeting a critical bug.

The codemod does not support projects that doesn't have a `dependency` key in `package.json` currently. It throws a `TypeError: Cannot read property 'ember-component-css' of undefined`. Newly created Ember applications don't have a `dependency` key in their `package.json`. Actually I needed to add that key to the projects in order to be able to test this codemod...

Using `path.join()` instead of doing an assumption of platform-specific separator also seems like a good fix to me even so I haven't used this Codemod on Windows. I think this needs to be done at some other places (e.g. [here](https://github.com/ember-codemods/tagless-ember-components-codemod/blob/da992e5351f5cdacf0af74f745946b2793590614/lib/transform.js#L324) and [here](https://github.com/ember-codemods/tagless-ember-components-codemod/blob/da992e5351f5cdacf0af74f745946b2793590614/lib/transform.js#L327) as well. But that shouldn't be a blocker for merging this fix, is it?

File: lib/index.js [hunk 1]
```diff
@@ -39,7 +40,7 @@ async function runForGlobs(patterns, cwd, options = {}) {
   debug('componentPaths = %O', paths);
 
   let packageRoot = await pkgDir(cwd);
-  let pkgContent = fs.readFileSync(`${packageRoot}/package.json`);
+  let pkgContent = fs.readFileSync(fsPath.join(packageRoot, 'package.json'));
```
Comment (Turbo87) [inline lib/index.js]: AFAIK the `fs.readFileSync()` API takes care of this issue internally already. are you sure that this is needed?

**Comment by Turbo87** [comment]
closing due to inactivity

**Comment by Alonski** [comment]
@Turbo87 I'd still like this to be merged :)

**Comment by Turbo87** [comment]
why?

**Comment by Alonski** [comment]
Because this wasn't working for me on Windows 😅

**Comment by Turbo87** [comment]
as I mentioned above, my understanding was that Node internally takes care of these things. what error exactly are you seeing?

I think if we want to fix this then we should probably first work on #54, then add Windows testing to CI and then fix the resulting issues. if we don't have testing for this then it's very easy to regress on Windows support here.

**Pull request closed without merging**